.travis.yml

```diff
@@ -18,6 +18,7 @@ bundler_args: --with integration --without changelog debug docs
 script:
   - bundle exec rake
   - export KITCHEN_YAML=kitchen.dokken.yml
+  # - export CHEF_LICENSE=accept-no-persist
```

Comment:
Looks like tests are failing without this

Reply:
:doh: this was a local change I made to test and checked it in